int64 baudrate
string port
string device_name
---
bool result